src/nibiru/index.ts

```diff
@@ -3,46 +3,17 @@ import type { LocalConfig } from '@/stores';
 const PLAYGROUND_NETWORKS = 'https://networks.play.nibiru.fi/ping-pub';
 const DEV_NETWORKS = 'https://networks.testnet.nibiru.fi/ping-pub';
 const ITN_NETWORKS = 'https://networks.itn.nibiru.fi/ping-pub';
+const MAIN_NETWORK = 'https://networks.nibiru.fi/ping-pub';
 
-export const getItn = async (): Promise<LocalConfig[] | undefined> => {
+export const getNetwork = async (
+  url: string
+): Promise<LocalConfig[] | undefined> => {
```

Comment:
Might be cleaner to have this function only return `LocalConfig[]`, that way you don't need to do the ?? check in the calling code.